src/components/sound.js

```diff
@@ -22,35 +23,39 @@ module.exports.Component = registerComponent('sound', {
   init: function () {
     this.listener = null;
     this.audioLoader = new THREE.AudioLoader();
-    this.sound = null;
+    this.soundsPool = new THREE.Group();
```

Comment:
This could be `this.pool` (since we're already in the sound component) or `this.soundPool` (since `pool` already connotes plural)